ads/_a4a-config.js

```diff
@@ -84,3 +84,22 @@ export const signingServerURLs = {
   'cloudflare': 'https://amp.cloudflare.com/amp-ad-verifying-keyset.json',
   'cloudflare-dev': 'https://amp.cloudflare.com/amp-ad-verifying-keyset-dev.json',
 };
+
+/** @type {!../extensions/amp-a4a/0.1/refresh-manager.RefreshConfig} */
+export const DEFAULT_REFRESH_CONFIG = {
+  visiblePercentageMin: 50,
+  totalTimeMin: 0,
```

Comment:
Total time the ad must be on screen to be considered "seen". This is the same parameter that amp-analytics' visibility takes.